src/main/java/codeu/model/data/Conversation.java

```diff
@@ -61,4 +76,53 @@ public String getTitle() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  /** Returns the list of users that can access and chat in this Conversation */
+  public List<User> getUsers() {
+    return users;
+  }
+
+  /** Returns the number of users that can access and chat in this Conversation*/
+  public int getNumUsers() {
+    return users.size();
+  }
+
+  /** Adds a user to the user List by using their username
+   * @param username Username of the user to add
+   * @return boolean whether or not the user was found and added into the List */
+  public boolean addUser(String username) {
+    if (username == null) return false;
+
+    // TODO: refactor this to make it work with tests(originally used this to add users in ProfilePageServlet doPost)
+    User user = UserStore.getInstance().getUser(username);
+    if (user != null) {
+      users.add(user);
+      return true;
+    }
+
+    return false;
+  }
+
+  /** Returns whether or not this Conversation is a normal conversation Direct Message */
+  public boolean isNormalConversation() {
```

Comment:
Why not be more explicit with this. You could have an Enum `ConversationType` that could be one of `DIRECT`, `NORMAL` or `GROUP`. Will make checking less reliant on implicit assumptions that may change in the future

Reply:
Thanks for the suggestion, this should really help my code look way more clean and more easily extensible when I go on to do the MVP. Just added this in the new commit

Reply:
Just a quick question; is using static imports to change"Conversation.ConversationType.NORMAL" it into a more succinct "ConversationType.NORMAL" okay or should I just change the name of enum to "Type" (since it's part of the Conversation class anyways?)